os/booting-on-vagrant.md

```diff
@@ -50,20 +50,56 @@ coreos:
     listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
     listen-peer-urls: http://$private_ipv4:2380
   fleet:
-      public-ip: $public_ipv4
+    public-ip: $public_ipv4
+  flannel:
+    interface: $public_ipv4
   units:
     - name: etcd2.service
       command: start
     - name: fleet.service
       command: start
+    - name: flanneld.service
```

Comment:
Lots of people want complete solution. I will just leave this string without "command: stop"